oauth2/provider/login.php

```diff
@@ -61,7 +44,7 @@
                                 $value = 'True';
                             }
                             $key = ucwords(str_replace("_", " ", $key));
-                            echo "<label class='col-form-label'><b>" . $key . ":</b>  " . $value . "</label><br />\n";
+                            echo "<label class='col-form-label'><b>" . text($key) . ":</b>  " . text($value) . "</label><br />\n";
```

Comment:
what would a PR be if you didn't catch me not escaping!! :)

Reply:
yep, it reminded of the old times :)